Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -591,6 +581,7 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 }
             }
             _analyzer = analyzer;
+            TryToAcquireCurrentTypings();
```

Comment:
This function is internally gated to only download the typings if IntelliSense is ES6 and a number of other conditions are met.